### PR TITLE
[DOCS] Option to disable device type extraction in user agent processor

### PR DESCRIPTION
**Pull request by danhermann**
Depends on #71625.

Doc preview: https://elasticsearch_71626.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/user-agent-processor.html

### CODE REVIEW

**Comment by danhermann** [comment]
@elasticmachine update branch

File: docs/reference/ingest/processors/user-agent.asciidoc [hunk 1]
```diff
@@ -21,6 +21,7 @@ The ingest-user-agent module ships by default with the regexes.yaml made availab
 | `target_field`         | no        | user_agent                                                                                      | The field that will be filled with the user agent details.
 | `regex_file`           | no        | -                                                                                               | The name of the file in the `config/ingest-user-agent` directory containing the regular expressions for parsing the user agent string. Both the directory and the file have to be created before starting Elasticsearch. If not specified, ingest-user-agent will use the regexes.yaml from uap-core it ships with (see below).
 | `properties`           | no        | [`name`, `major`, `minor`, `patch`, `build`, `os`, `os_name`, `os_major`, `os_minor`, `device`] | Controls what properties are added to `target_field`.
+| `extract_device_type`  | no        | `false`                                                                                         | (beta) Extracts device type from the user agent string on a best-effort basis
```
Comment (jrodewig) [inline docs/reference/ingest/processors/user-agent.asciidoc:24]: I'd use the beta admon to get the auto-generated text on rollover.
Screen Shot 2021-04-19 at 9 31 35 AM

I'd also add a period for consistency.

```suggestion
| `extract_device_type`  | no        | `false`                                                                                         | beta:[] Extracts device type from the user agent string on a best-effort basis.
```

**Review by jrodewig — APPROVED** [review]
Thanks for your patience on this review @danhermann. I was away on vacation for a bit.

LGTM with a couple of minor changes.

**Comment by danhermann** [comment]
> Thanks for your patience on this review @danhermann. I was away on vacation for a bit.

Thanks, @jrodewig. I saw that you were on vacation and I hope it was a good one for you!

**Pull request merged**